src/index.html

```diff
@@ -3,12 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a class="logo_link" href="#">
+        <img class="logo"
+        src="images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li><a href="#" class="nav_link is-active">APPLE</a></li>
+          <li><a href="#" class="nav_link">SAMSUNG</a></li>
+          <li><a href="#" class="nav_link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+          <li>
+            <a href="#"
+            class="nav_link"
+            data-qa="hover">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+          </li>
```

Comment:
```suggestion
          <li>
            <a
              href="#"
              class="nav_link"
              data-qa="hover"
              >
                НОУТБУКИ И КОМПЬЮТЕРЫ
              </a>
          </li>
```
check your code style and do like this